library-scripts/docs/sshd-debian.md

```diff
@@ -31,12 +33,12 @@ Or as a feature:
 | New password for user |                | `skip`      | Sets a new password for the specified non-root user. A value of `skip` will skip this step. A value of `random` will generate a random password and print it out when the script finishes.                                                              |
 | Fix environment flag  |                | `true`      | Connections using the SSH daemon use "fresh" login shells. While Remote - Containers and Codespaces handle most environment variables automatically, Codespaces secrets require additional processing.                                                  |
 
-## Usage
+### Usage
 
 This script can be used as a feature, ad-hoc in an already running container, or
 in a Dockerfile.
 
-### Feature use
+#### Feature Use
```

Comment:
```suggestion
#### Feature Use
```

it is incredibly ironic that this feature is _still_ here; the reason why we forked their unmaintained bash script library is because they switched to only supporting features, which offer very little control and/or customization.

we are strong against features (at least i am; i think we should remove all of that), fwiw